x/mongo/driver/topology/sdam_spec_test.go

```diff
@@ -561,3 +561,169 @@ func TestSDAMSpec(t *testing.T) {
 		}
 	}
 }
+
+func TestHasStalePrimary(t *testing.T) {
+	t.Parallel()
+
+	t.Run("WV17 SE EQ FME and SSV LT FSMV", func(t *testing.T) {
```

Comment:
[nit] Haha love these acronyms but what exactly do they mean? If it's "wire version 17, server election ID equal to max election ID, and server set version less than max set version" maybe do "WV17 SEI EQ MEI and SSV LT MSV"?

Reply:
Sure, I can make that change! Thanks for the suggestion